src/programs/unblur/unblur.cpp

```diff
@@ -188,8 +191,8 @@ void UnBlurApp::DoInteractiveUserInput()
 	bool write_out_small_sum_image = false;
 	std::string small_sum_image_filename = "/dev/null";
 
-	my_current_job.Reset(37);
-	my_current_job.ManualSetArguments("ttfffbbfifbiifffbsbsfbfffbtbtiiiibtti",input_filename.c_str(),
+	my_current_job.Reset(38);
```

Comment:
BTW Reset is not needed ManualSetArguments does a Reset.

Reply:
OK thanks. I'll remove that line.